gputop/remote/ajax/metrics.html

```diff
@@ -212,16 +216,12 @@ <h2 id="page-header-metrics">Metrics panel loading</h2>
 
     function draw_filter_checkboxes()
     {
-        if (selected_index < 0)
-            return;
-
```

Comment:
I believe it was added in this commit; https://github.com/Kasi-Reddy/gputop/commit/0a6d63cdce068ea10955bd56ffe406e581c53e3f

Yeah, i can't see it happening. The map is indexed via metric names so checking against 0 would be redundant.